doc/src/modules/discrete.rst

```diff
@@ -0,0 +1,95 @@
+Discrete Module
+===============
+
+The ``discrete`` module in SymPy implements methods to compute discrete
+transforms and convolutions of finite sequences.
+
+.. automodule:: sympy.discrete
+
+Since the discrete transforms can be used to reduce the computational complexity
+of discrete convolutions, therefore, convolutions module makes use of transforms
```

Comment:
Remove "therefore" (it is redundant with "since"), add "the", and make `convolutions` code to differentiate it from the previous word, like "convolutions, the `convolutions` module makes use..."

Reply:
"the `transforms` module"